core/src/main/java/com/linecorp/armeria/server/ServiceRequestContext.java

```diff
@@ -159,7 +159,10 @@ static ServiceRequestContextBuilder builder(HttpRequest request) {
 
     /**
      * Returns the address of the client who initiated this request.
+     *
+     * @deprecated Use {@link #proxiedAddresses()} with {@link #clientAddressMapper()}
      */
+    @Deprecated
     default InetAddress clientAddress() {
```

Comment:
I think this method should not be deprecated because it is fairly often that a user wants to know about the actual client address when writing a webapp.

Could you make sure this method returns the correct client address calculated with `clientAddressMapper` and `proxiedAddresses`?